modules/distr-tracing-tempo-deploy-default.adoc

```diff
@@ -0,0 +1,115 @@
+
+[NOTE]
+====
+In-memory storage is not persistent. If the Tempo pod shuts down, restarts, or is replaced, your trace data will be lost. For persistent storage, you must use the `production` or `streaming` strategies, which use Elasticsearch as the default storage.
+====
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `tracing-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `tracing-system` from the *Project* menu. You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the {TempoName} Operator. On the *Details* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Tempo*, click *Create Instance*.
+
+. On the *Create Tempo* page, to install using the defaults, click *Create* to create the {TempoShortName} instance.
+
+. On the *Tempos* page, click the name of the {TempoShortName} instance, for example, `tempo-all-in-one-inmemory`.
+
+. On the *Tempo Details* page, click the *Resources* tab. Wait until the pod has a status of "Running" before continuing.
+
+
+[id="distr-tracing-deploy-default-cli_{context}"]
+== Deploying the {DTShortName} default strategy from the CLI
+
+Follow this procedure to create an instance of {TempoShortName} from the command line.
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed and verified.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the OpenShift CLI (`oc`) that matches your {product-title} version.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
++
+[source,terminal]
+----
+$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:8443
+----
+
+. Create a new project named `tracing-system`.
++
+[source,terminal]
+----
+$ oc new-project tracing-system
+----
+
+. Create a custom resource file named `tempo.yaml` that contains the following text:
++
+.Example tempo-all-in-one.yaml
+[source,yaml]
```

Comment:
We need to review this CR. Some required steps are missing. And the `apiVersion` is wrong